addons/website_sale/static/tests/tours/website_sale_google_analytics.js

```diff
@@ -51,12 +51,12 @@ tour.register('google_analytics_view_item', {
 
 tour.register('google_analytics_add_to_cart', {
     test: true,
-    url: '/shop?search=Acoustic Bloc Screens',
+    url: '/shop?search=Storage Box',
```

Comment:
The following commit changed the name of the product `product.product_product_25` from `Acoustic Bloc Screens` to `Laptop E5023 (IVA 10,5`, and since the tour is looking for the original name of the product and therefore not available, the tour fails. (commit: https://github.com/odoo/odoo/commit/ebe291560ac9c573cdd6e1b73fc9ea3c658ce070#diff-958b2f65570de219b51598626056199b5c945629c4033cc4aebecdd7b4956f82R90-R95 )
Instead, use the `Storage Box` product that is created before running the test tour.